### PR TITLE
Remove liquid token converter

### PR DESCRIPTION
**Pull request by barakman**

### CODE REVIEW

File: solidity/test/Converter.js [hunk 1]
```diff
@@ -220,7 +186,7 @@ describe('Converter', () => {
         reserveToken2 = await TestNonStandardToken.new('ERC Token 2', 'ERC2', 18, 2000000000);
     });
 
-    for (const type of [0, 1, 3, 4]) {
+    for (const type of [1, 3, 4]) {
```
Comment (lbeder) [inline solidity/test/Converter.js:189]: Let's all play along with that `2` doesn't exist 😅

**Review by lbeder — APPROVED** [review]

File: solidity/test/Converter.js (hunk 1 above)
Comment (barakman) [reply solidity/test/Converter.js:189]: How exactly do you wanna handle this?

Comment (lbeder) [reply solidity/test/Converter.js:189]: I was joking. Nothing to do here 😎

File: solidity/test/ConverterFactory.js [hunk 2]
```diff
@@ -4,13 +4,19 @@ const { expect } = require('../../chai-local');
...
 const StandardPoolConverterFactory = contract.fromArtifact('StandardPoolConverterFactory');
+const FixedRatePoolConverterFactory = contract.fromArtifact('FixedRatePoolConverterFactory');
 const TypedConverterAnchorFactory = contract.fromArtifact('TestTypedConverterAnchorFactory');
 const ConverterBase = contract.fromArtifact('ConverterBase');
 const DSToken = contract.fromArtifact('DSToken');
 
+const Factories = {
+    LiquidityPoolV1ConverterFactory,
+    StandardPoolConverterFactory,
+    FixedRatePoolConverterFactory
```
Comment (yudilevi) [inline solidity/test/ConverterFactory.js:17]: fixed rate converter shouldn't be here

File: solidity/test/ConverterRegistry.js [hunk 3]
```diff
@@ -664,15 +652,13 @@ describe('ConverterRegistry', () => {
             anchorC = await DSToken.new('TokenC', 'TKNC', 18);
             anchorE = await DSToken.new('TokenE', 'TKNE', 18);
 
-            converter1 = await LiquidTokenConverter.new(anchor1.address, contractRegistry.address, 0);
             converter2 = await LiquidityPoolV1Converter.new(anchor2.address, contractRegistry.address, 0);
```
Comment (yudilevi) [inline solidity/test/ConverterRegistry.js:655]: names should be updated (or add a new `converter1`)

**Review by yudilevi — CHANGES_REQUESTED** [review]

File: solidity/test/ConverterFactory.js (hunk 2 above)
Comment (barakman) [reply solidity/test/ConverterFactory.js:17]: This is the only place in our code which utilizes and tests contract `FixedRatePoolConverterFactory`.

Comment (yudilevi) [reply solidity/test/ConverterFactory.js:17]: Ok, fair enough

File: solidity/test/ConverterRegistry.js (hunk 3 above)
Comment (barakman) [reply solidity/test/ConverterRegistry.js]: Done.

**Review by yudilevi — APPROVED** [review]

**Pull request merged**